src/components/CopyButton.tsx

```diff
@@ -10,7 +10,7 @@ const CopyButton = ({ data, label }) => {
     const [buttonClass, setButtonClass] = createSignal<string>("btn");
     const [buttonActive, setButtonActive] = createSignal<boolean>(false);
     const onClick = () => {
-        clipboard(data);
+        clipboard(data.replaceAll(" ", ""));
```

Comment:
What if we ever want to copy something that needs to contain a space? Could we only apply this to numbers?

Reply:
right now spaces are never valid and `50 000` is also not a number